src/category_theory/monoidal/functor.lean

```diff
@@ -0,0 +1,183 @@
+-- Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Michael Jendrusch, Scott Morrison
+import category_theory.monoidal.category
+
+open category_theory
+open tactic
+
+universes v₁ v₂ v₃ u₁ u₂ u₃
+
+open category_theory.category
+open category_theory.functor
+
+namespace category_theory
+
+section
+
+open monoidal_category
+
+variables (C : Sort u₁) [𝒞 : monoidal_category.{v₁} C]
+          (D : Sort u₂) [𝒟 : monoidal_category.{v₂} D]
+include 𝒞 𝒟
+
+structure lax_monoidal_functor extends C ⥤ D :=
+-- unit morphism
+(ε               : tensor_unit D ⟶ obj (tensor_unit C))
+-- tensorator
+(μ                : Π X Y : C, (obj X) ⊗ (obj Y) ⟶ obj (X ⊗ Y))
+(μ_natural'       : ∀ (X Y X' Y' : C)
```

Comment:
I would expect implicit arguments  `{X Y X' Y' : C}` here--normally we do this when introducing the endpoints of ensuing morphisms.

Reply:
Fixed.